functionaltests/src/test/java/specs/paymentmethods/BaseNotifiablePaymentFixture.java

```diff
@@ -74,6 +81,29 @@ public boolean receivedNotificationOfActionFor(final String paymentNames, final
         return success;
     }
 
+    /**
+     * Used to wait for second notification (paid) if the previous one (appointed) has been successful.
+     * @param paymentNames
+     * @param txaction current txAction to wait (paid)
```

Comment:
`txaction` is a custom Payone response field, it is described in README and Payone documentations. 

`txAction` camel-case has been used since the beginning of the project as a name for CTP custom field, hence kind of too late to change it back...